doc/nrf/protocols/bt_mesh/fota.rst

```diff
@@ -1,79 +1,17 @@
 .. _ug_bt_mesh_fota:
 
-Performing FOTA updates in Bluetooth mesh
-#########################################
+Performing Device Firmware Updates (DFU) in Bluetooth mesh
+##########################################################
 
-.. contents::
-   :local:
-   :depth: 2
+The |NCS| supports two DFU methods when updating your firmware over-the-air (FOTA) for your Bluetooth mesh devices and applications.
 
-In |NCS|, there is currently no support for Device Firmware Update (DFU) over Bluetooth® mesh when performing firmware over-the-air (FOTA) updates for your Bluetooth mesh devices and applications.
-However, support for FOTA updates with point-to-point DFU over Bluetooth Low Energy using the MCUmgr subsystem and the Simple Management Protocol (SMP) is available.
+* DFU over Bluetooth® mesh using the Zephyr Bluetooth mesh DFU subsystem that implements the Mesh Device Firmware Update Model specification version 1.0 and Mesh Binary Large Object Transfer Model specification version 1.0.
+  The specification that the Bluetooth mesh DFU subsystem is based on is not adopted yet, and therefore this feature should be used for experimental purposes only.
```

Comment:
I think I'd like this to explicitly say that this means it cannot be qualified and therefore not legally used in an end product you sell.

Reply:
We are using the term 'experimental' throughout the documentation. My understanding is that, it means, such feature cannot be used in production. If we start adding specifics about qualifications, we should add them at all places where experimental Bluetooth features are documented. 

What do you think?